test/interpolations.jl

```diff
@@ -46,7 +46,7 @@ end
 
     etp = @inferred ImageTransformations.box_extrapolation(img)
     @test @inferred(ImageTransformations.box_extrapolation(etp)) === etp
-    @test_broken summary(etp) == "2×2 extrapolate(interpolate(::Array{Gray{N0f8},2}, BSpline(Linear()), OnGrid()), Gray{N0f8}(0.0)) with element type $(ctqual)Gray{$(fpqual)Normed{UInt8,8}}"
+    @test summary(etp) == "2×2 extrapolate(interpolate(::Array{Gray{N0f8},2}, BSpline(Linear()), OnGrid()), Gray{N0f8}(0.0)) with element type $(ctqual)Gray{$(fpqual)Normed{UInt8,8}}"
```

Comment:
isn't this still broken until https://github.com/JuliaMath/Interpolations.jl/pull/226 is merged?

Reply:
Working on it now. Hoping to have it finished by tomorrow.

Reply:
Your'e right, I was testing with the `showarg` fixes in place.  I'll set this to "skipped" like the others for now.